src/sentry/tasks/digests.py

```diff
@@ -18,8 +19,8 @@
     queue="digests.scheduling",
     silo_mode=SiloMode.REGION,
 )
-def schedule_digests():
-    from sentry import digests
+def schedule_digests() -> None:
+    from sentry.digests import backend as digests
```

Comment:
I would recommend changing to `digests.backend.whatever(...)` instead rather than the import renaming

this will make it easier to undo later if we can make a plugin for the service delegator pattern